server/notification-providers/pushover.js

```diff
@@ -43,15 +44,19 @@ class Pushover extends NotificationProvider {
             if (heartbeatJSON == null) {
                 await axios.post(url, data);
                 return okMsg;
-            } else {
-                data.message += `\n<b>Time (${heartbeatJSON["timezone"]})</b>:${heartbeatJSON["localDateTime"]}`;
-                await axios.post(url, data);
-                return okMsg;
             }
+
+            if (heartbeatJSON.status === UP) {
```

Comment:
In the current implementation, previous pushover users might NOT get a sound for these notifications or cause an error, depending on how pushover handles `null`.
Let's check for its existance

```suggestion
            if (heartbeatJSON.status === UP && notification.pushoversounds_up) {
```